content/pricing/free.textile

```diff
@@ -55,6 +55,39 @@ To downgrade your account to a Free package:
 2. Log in to your "account":https://ably.com/login and select *Billing* from the *Account* menu.
 3. Click the *Downgrade* button.
 
-If you downgrade to a Free package, you will finish the month on your current package with its associated limits and benefits. At the beginning of the following month you will receive a final invoice covering your last month's consumption and package fee. Once the final invoice has been paid, your payment details are removed from the system.
+If you downgrade from a Standard or Pro package, you will finish the month on the Standard package with its associated limits and benefits. At the beginning of the following month you will receive a final invoice covering your last month's consumption. Once the final invoice has been paid, your payment details are removed from the system.
+
+<aside data-type='note'>
+<p>Be aware that this will permanently delete all the information associated with your account.</p>
+</aside>
+
+h2(#close). Close your account
+
+You can close your account at any time by first "downgrading":#downgrade to the "Free package":/docs/pricing/free. The following steps outline the the process to close your account:
+
+* Ensure you are the account "owner":/docs/account/users#roles.
+* Downgrade all paid "packages":/docs/pricing#packages free packages:
+** Navigate to *Account* then *Billing*.
+** Click *Save billing details* to save your changes.
+* After all packages have been downgraded to *Free*, go to the *My Settings* page.
+** Navigate to *Want to delete your profile?*.
+** Click *start* to begin the closure process.
+* On the *Close Your Ably Account* page:
+** Review the account summary, including ownership, plan type, and app details.
+** Confirm all accounts are on the *Free* tier and details are accurate.
+** Click *Confirm Closure* to permanently deactivate your Ably account.
+
+If you sign in with SSO, for example, Google or GitHub, you cannot use the self-service account closure process. To proceed, you must set a password and disconnect their SSO provider via My Settings.
+
+* Log in to your Ably account using your current SSO method (Google or GitHub).
+* Navigate to *Account* then *My Settings*.
+* In the *Password* section, click *Change your password*.
+* Click *Update my personal settings* to save the changes.
+* Scroll to the *Login provider* section.
+* Click *remove connection* next to the SSO provider/s you want to disconnect.
+
+<aside data-type='note'>
+<p>Closing your account will permanently delete all associated data and cannot be undone.</p>
+</aside>
```

Comment:
I'd suggest the same as per above about not repeating all of this info on each page. My reasoning for this is that it's quite a large number of steps compared to the upgrade/downgrade process.